docs/sandboxing/session-id/proposal.md

```diff
@@ -0,0 +1,53 @@
+# Session ID Security Proposal
+
+This proposal seeks to sandbox tokens used to log in to Minecraft servers, which are called session IDs.
+
+## Background
+
+Currently, session IDs are stored on the Minecraft client. The client presents the session IDs to Mojang's session server when logging into a server. This configuration presents several issues when it comes to security. A malicious Minecraft mod could potentially read the session IDs from the process, and send it to its owner. This would allow the malware spreader to use the victim's session IDs to log in to Minecraft servers for 24 hours. There is no way to invalidate the session ID, and the only recourse the victim has is to disable multiplayer in their Xbox settings.
+
+## Proposed solution
+
+For clarity, we will refer to Java process that runs Minecraft as the Minecraft process, and the other process as the session ID process.
```

Comment:
What does "the other process" refer to? I don't think this is explicit enough to clarify.